app/assets/stylesheets/components/_form.scss

```diff
@@ -16,3 +16,13 @@
   margin-left: .5rem;
   vertical-align: baseline;
 }
+
+input[type='number'] {
+  -moz-appearance: textfield;
+}
+
+
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+}
```

Comment:
Sorry, I should have included a screenshot in the PR for that. Changing it to a number field caused desktop browsers to add a dial to the field which I assume we do not want.